lib/ret_web/controllers/page_controller.ex

```diff
@@ -198,38 +194,49 @@ defmodule RetWeb.PageController do
     end
   end
 
-  defp get_configurable_asset(cache_key, config_key, fallback_file) do
-    case Cachex.get(:assets, cache_key) do
-      {:ok, nil} ->
-        app_config = AppConfig |> Repo.get_by(key: config_key) |> Repo.preload(:owned_file)
+  defp respond_with_configurable_asset(conn, cache_key, config_key, content_type) do
```

Comment:
as a sanity check, did you verify we do no database queries when the cache is warm?

Reply:
yup, no queries when the cache is warm.